packages/aws-client/src/types.ts

```diff
@@ -36,6 +34,4 @@ export type AWSClientTypes =
     CloudWatchEvents |
     CloudFormation |
     Kinesis |
-    KMS |
-    AWS.Service |
-    AWS.DynamoDB.DocumentClient;
+    KMS;
```

Comment:
Were we just not using these?

Reply:
`AWS.Service` wasn't used and `AWS.DynamoDB.DocumentClient` became

```
export const dynamodbDocClient = (docClientOptions?: TranslateConfig, dynamoOptions?: object) =>
  DynamoDBDocument.from(
    awsClient(DynamoDB, '2012-08-10')(dynamoOptions),
    docClientOptions
  );
 ```